docs/usage/custom_cards/custom_card_device_tracker.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Device Tracker Custom-card
```

Comment:
Are the changes to the device_tracker custom card within this merge request intentional?